KhrystynaVehera/HW5/Task2.py

```diff
@@ -0,0 +1,11 @@
+users_number = int(input("Please enter number to switch Fibonacci number: "))
```

Comment:
good job
but I recommend using more meaningful variable names
look at this version of code
n = int(input('enter some number from fibonacci list= '))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])
print(f'there are some numbers from fibanacci till that you entered {n},\n {fibonacci_numbers}')